app/components/audio/song-control.tsx

```
@@ -114,22 +114,24 @@ export default function SongControl() {
-    )}>
-      <div className="p-4 gap-4 flex flex-wrap items-center justify-between">
+    )} onClick={() => {
+      if (window.innerWidth < 768 && currentSong?.id) router.push(`/song/${currentSong.id}`)}
+    }>
+      <div className="p-4 gap-1 md:gap-4 flex flex-wrap items-center justify-between">
         <div className="song-title flex items-center gap-2 w-fit md:w-1/4">
           <div className={twMerge(
             "relative cursor-pointer",
             (isLoading || isEmpty) && "opacity-50",
             isLoading && "animate-pulse"
           )} onClick={handleImageClick}>
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)